ebml_matroska.xml

```diff
@@ -130,7 +130,7 @@
   <element name="BlockAdditional" path="1*1(\Segment\Cluster\BlockGroup\BlockAdditions\BlockMore\BlockAdditional)" id="0xA5" type="binary" minOccurs="1" maxOccurs="1" minver="1" webm="0">
     <documentation lang="en">Interpreted by the codec as it wishes (using the BlockAddID).</documentation>
   </element>
-  <element name="BlockDuration" path="0*1(\Segment\Cluster\BlockGroup\BlockDuration)" id="0x9B" type="uinteger" maxOccurs="1" minver="1" default="DefaultDuration">
+  <element name="BlockDuration" path="0*1(\Segment\Cluster\BlockGroup\BlockDuration)" id="0x9B" type="uinteger" maxOccurs="1" minver="1" default="\Segment\Tracks\TrackEntry\DefaultDuration">
```

Comment:
This is an improvement, but still it's unclear *which* TrackEntry is referred to.
With an xpath style reference would be like:
`\Segment\Tracks\TrackEntry[TrackNumber=./Block/TrackNumber]\DefaultDuration`
to link the intended TrackEntry to the TrackNumber in the Block Header; unfortunately this style of xpath linking isn't feasible since the Track Number of the Block Header is not an element but a byte range of binary data.